From 6964b6b6839ef5db63fbfacdd721ae58a75c70a8 Mon Sep 17 00:00:00 2001
From: DBAA <hellocaoziyi@gmail.com>
Date: Wed, 27 May 2020 11:33:20 +0800
Subject: [PATCH 26/32] Update qtgui_chooser.block.yml

Trying to build a chooser block fails with Generate Error: (NameError("'i' is not defined")
154b2b7 commit is missing from maint-3.8.
---
 gr-qtgui/grc/qtgui_chooser.block.yml | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/gr-qtgui/grc/qtgui_chooser.block.yml b/gr-qtgui/grc/qtgui_chooser.block.yml
index df27f02468..8e9efd0513 100644
--- a/gr-qtgui/grc/qtgui_chooser.block.yml
+++ b/gr-qtgui/grc/qtgui_chooser.block.yml
@@ -126,11 +126,7 @@ templates:
         % if int(num_opts):
         self._${id}_labels = (\
         % for lbl in all_labels:
-        % if lbl:
         ${lbl}, \
-        % else:
-        self._${id}_options[${i}], \
-        % endif
         % endfor
         )
         % elif labels:
-- 
2.20.1

